be/src/olap/storage_engine.h

```diff
@@ -225,7 +224,7 @@ class StorageEngine {
 
     OLAPStatus _judge_and_update_effective_cluster_id(int32_t cluster_id);
 
-    OLAPStatus _start_bg_worker();
+    bool _too_many_disks_are_broken(uint32_t unused_num, uint32_t total_num);
```

Comment:
```suggestion
    bool _check_disks_broken_percentage(uint32_t unused_num, uint32_t total_num);
```

Reply:
Ummm, coincidentally, I did take some time to name it. Not to be pedantic, but I do think the original is better.  :-)

The method's return type is `bool`, return `true` means: "yes, it is too many, **so you should do something(the process will exit)".**

so, finally, I choosed from these two: 
1. `_are_too_many_disks_broken()`
2. `_too_many_disks_are_broken()`

On the other hand, `percentage` is a neutral noun(although we do judge by percentage), returning true, can not directly indicate whether it is a high or a low percentage,  or whether we expect it to be high or low. And like some other `check_xxx()`methods, It may be misinterpreted as: when `true` is returned, it is a valid situation (**so we don't need to do anything**).

but if we choose `_check_disks_broken_percentage_is_too_hign()`, it would be too long.

Reply:
OK